kimchi/src/circuits/gate.rs

```diff
@@ -91,11 +94,48 @@ pub enum GateType {
     CairoInstruction = 13,
     CairoFlags = 14,
     CairoTransition = 15,
-    // Range check (16-24)
+    /// Range check (16-24)
```

Comment:
For some reason we were leaving the numbers between 16 and 24 for range check and foreign arithmetics. Taking into account the latest version of these gates, we are taking from 16 to 19. Matthew wanted to leave that comment as is, so I haven't changed the interval just in case.

Reply:
I think it's fine to reserve these ranges, but if we do that then ffmul and ffadd should start at 25 no?

Reply:
Yeah that's what I thought... I was not sure what was going to be the standard here. Perhaps Matthew had some ideas that did not include the FFMul and FFAdd gates. If this is the case, then I will start ffmul and ffadd in 25 just in case. Good catch.